doc/source/Tutorials/Scripting/QueryingAScene/index.md

```diff
@@ -1,296 +1,135 @@
-Querying A Scene
-================
+# Tutorial: Querying a Scene #
 
-Gaffer has a simple API for querying the scenes that are output from nodes. In this tutorial we'll see how to use this API to traverse a scene hierarchy and examine its state.
+Gaffer's scripting flexibility allows you to query the scene's parameters and objects as it flows through the node graph. In this tutorial, we will demonstrate how to use Gaffer's API to reference various scene parameters and traverse the scene hierarchy. We will cover the following:
 
-Making a scene
---------------
+- The main `out` plug
+- Referencing the `globals` scene plug and its values
+- Using the utility methods for returning various scene properties:
+    - `object()` method, for returning a scene object
+    - `parameters()` method, for returning an object's scene parameters
```

Comment:
Let's omit this one from this list - the other's are all primary properties of a location, whereas this one is more esoteric, and limited to cameras.

Reply:
Should I remove the section on using `parameters()` to return camera parameters, too?